cypress/integration/seededFlows/adminFlows/users/manageCredits.spec.js

```diff
@@ -14,7 +14,7 @@ describe('Manage User Credits', () => {
       cy.testSetup();
       cy.fixture('users/adminUser.json').as('user');
       cy.get('@user').then((user) => {
-        cy.loginAndVisit(user, '/admin/users');
+        cy.loginAndVisit(user, '/admin/member_manager/users');
```

Comment:
I'd love to use `js-routes` in these files to avoid having to make route changes like these but I think we had an issue with it the last time we used it. Hence, I plan on adding an issue to our board and maybe during cooldown week I can look into it.